services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/switches/SwitchRulesSpec.groovy

```diff
@@ -457,4 +518,20 @@ class SwitchRulesSpec extends BaseSpecification {
 
         return rules
     }
+
+    void checkTrafficCountersForRules(Switch sw, FlowEndpointPayload flowEndpoint) {
```

Comment:
I think it'd be nice to add boolean parameter "isTrafficExpected". This way you can also use this function on line 472 instead of duplicating the code.